lib/v2/screens/profile_screens/contribution/contribution_screen.dart

```diff
@@ -47,7 +47,7 @@ class _ContributionScreenState extends State<ContributionScreen> with TickerProv
     return Scaffold(
       appBar: AppBar(title: Text('Contribution Score'.i18n)),
       body: BlocConsumer<ContributionBloc, ContributionState>(
-        cubit: _contributionBloc,
+        bloc: _contributionBloc,
```

Comment:
@RaulUrtecho  can u confirm this change is good

Reply:
let guess Bloc 7.0?  I belive its the same just a rename for this param